www/js/controller.js

```diff
@@ -493,4 +494,20 @@ module.exports = {
   autoplay() {
     global.core.search.checkAutoPlay();
   },
+
+  livefeed(val) {
+    if (val) {
+      dialog.showOpenDialog({
+        defaultPath: remote.app.getPath('desktop'),
+        properties: ['openDirectory'],
+      }, (path) => {
+        store.set('userPrefs.app.live-feed-location', path[0]);
+        const locationLabel = document.getElementsByClassName('sub-label livefeed');
+
+        for (let i = 0, len = locationLabel.length; i < len; i += 1) {
```

Comment:
How many places is it supposed to update? Is a `for` loop necessary?

Reply:
At the moment 2 places (display tab and setting pane), when https://github.com/KhalisFoundation/sttm-desktop/issues/280 is done, it should be one place, can refactor once that change is in. 

for loop might be much but it gets the job done.